docs/Operators.md

```diff
@@ -4632,7 +4632,7 @@ Other versions of this operator: <a href="Changelog.md#DequantizeLinear-10">10</
 
 <dl>
 <dt><tt>axis</tt> : int (default is 1)</dt>
-<dd>(Optional) The axis of the dequantizing dimension of the input tensor. Negative value means counting dimensions from the back. Accepted range is [-r, r-1] where r = rank(input)</dd>
+<dd>(Optional) The axis of the dequantizing dimension of the input tensor. Negative value means counting dimensions from the back. Accepted range is [-r, r-1] where r = rank(input). If either y_zero_point or y_scale is a 1D tensor, then an axis must be specified. If both are scalars, axis is ignored.</dd>
```

Comment:
I like the line `If both are scalars, axis is ignored.`.  The line `If either y_zero_point or y_scale is a 1D tensor, then an axis must be specified.` seems to conflict with axis having a default value (and technically being 1 if it is unspecified).

Reply:
Good catch. My fault, I didn't notice the default value and suggested a more complete description to Jacky.

Reply:
I actually wish the default value didn't exist so we could do it your way but unfortunately it's too late to change I think.

Reply:
Just removed `If either y_zero_point or y_scale is a 1D tensor, then an axis must be specified.` to prevent confusion. No worries @gramalingam  I didn't notice it in the first place and thank @TomWildenhain-Microsoft for catching it!

Reply:
Just removed If either y_zero_point or y_scale is a 1D tensor, then an axis must be specified. to prevent confusion. No worries @gramalingam I didn't notice it in the first place and thank @TomWildenhain-Microsoft for catching it!

Reply:
Minor nit: we should change ```both``` to ```both y_zero_point and y_scale``` since we deleted the preceding line. Otherwise, it is confusing what ```both``` refers to.

Reply:
Updated. Thanks!